llvm/include/llvm/IR/Function.h

```diff
@@ -942,9 +942,14 @@ class LLVM_ABI Function : public GlobalObject, public ilist_node<Function> {
   ///
   void viewCFG() const;
 
+  /// viewCFG - This function is meant for use from the debugger. It works just
+  /// like viewCFG(), but generates the dot file with the given filename.
+  void viewCFG(const char *Name) const;
+
   /// Extended form to print edge weights.
   void viewCFG(bool ViewCFGOnly, const BlockFrequencyInfo *BFI,
-               const BranchProbabilityInfo *BPI) const;
+               const BranchProbabilityInfo *BPI,
+               const char *Name = nullptr) const;
```

Comment:
nit: `Name` -> `outputFileName` to be explicit.

Reply:
Thanks for the review!
I've changed it to OutputFileName now.